src/view/com/util/post-embeds/ExternalPlayerEmbed.tsx

```diff
@@ -256,6 +256,16 @@ export function ExternalPlayer({
               ]}
             />
           </>
+        ) : (
+          <MediaInsetBorder
+            opaque
+            style={[
+              {
+                borderBottomLeftRadius: 0,
+                borderBottomRightRadius: 0,
+              },
+            ]}
+          />
```

Comment:
could just move this outside (below) the conditional render and remove the one up top if you want.